.circleci/config.yml

```diff
@@ -877,11 +867,6 @@ workflows:
           resource_class: medium
           requires:
             - docker-build-amd64
-      - report:
-          requires:
-            - acceptance-tests-amd64
-            - acceptance-tests-arm64
-            - unit-tests
```

Comment:
FYI: This is the important part here, we completely remove the report step from the workflow definition of the `acceptance-only-run` workflow.